test/ledger.integration.test.js

```diff
@@ -77,7 +77,6 @@ test('integration : v2 contribution workflow with uphold BAT wallet', async t =>
   t.true(response.body.hasOwnProperty('verification'))
 
   t.true(response.body.wallet.addresses.hasOwnProperty('BAT'))
-  t.true(response.body.wallet.addresses.hasOwnProperty('BTC'))
```

Comment:
these are deposit addresses on a card denominated in BAT.
we still want users to be able to deposit BTC which will be autoconverted by uphold to BAT, so we should still address that this address is created

Reply:
cool, i can put these marked lines back in.